docs/release-notes/2.37.0.md

```diff
@@ -0,0 +1,170 @@
+
+1.	Navigate to the **Main** menu and hover on the {{icon.dashboards}} _Dashboards_ icon.
+2.	Click **New folder**.
+3.	Provide a name for your folder, and then select **Create**.
+4.	Navigate to {{icon.dashboards}} _Dashboards_ from the **Main** menu and click **Browse**.
+5.  Select the dashboard that you want to move and click **Move**.
+6.	On the **Choose Dashboard** dialogue box, from the dropdown under **Folder** option, choose the folder where you want to move the dashboard.
+7.	To apply your changes, select **Save Dashboard**.
+
+
+## New Features
+
+- [PMM-10913](https://jira.percona.com/browse/PMM-10913) - Starting with PMM 2.37.0, you can use an external PostgreSQL server as a data storage for PMM. Thus, even if PMM fails, your data will be stored in an external source, reducing the risk associated with data loss. Furthermore, clustered PostgreSQL instances also enhance performance.
+
+- [PMM-11281](https://jira.percona.com/browse/PMM-11281) - We have written a query for the postgres exporter to deal with lock conflicts in databases that block several transactions, thus impacting performance. With this query, you can find the PID of the blocking session and how many sessions it blocked. The proactive approach enables customers to address non-responsive databases quickly by understanding the underlying causes.
+
+- [PMM-11384](https://jira.percona.com/browse/PMM-11384) - PMM 2.37.0 now supports PSMDB operator version 1.13.
+
+- [PMM-11438](https://jira.percona.com/browse/PMM-11438) - Starting with PMM 2.37.0, PMM administrators can disable the internal PostgreSQL server, which helps to fine-tune the server.
+
+- [PMM-11439](https://jira.percona.com/browse/PMM-11439) - Starting with PMM 2.37.0, a PMM administrator can now view metrics from an external PostgreSQL server.
+
+- [PMM-11675](https://jira.percona.com/browse/PMM-11675) - Telemetry with external PostgreSQL
+
+- [PMM-11834](https://jira.percona.com/browse/PMM-11834) - PMM 2.37.0 now supports PSMDB 1.14 operator.
+
+
+
+## Improvements
+
+- [PMM-4466](https://jira.percona.com/browse/PMM-4466) - We have migrated Grafana to use PostgreSQL instead of SQLite, which enhances the performance of the Auth system and sessions and makes the system less complex.
+- [PMM-11412](https://jira.percona.com/browse/PMM-11412) - We have enhanced our dashboards to be more user-oriented, displaying the DB cluster's status along with its components, such as metrics of stateful, pods, and so on. Also, you can apply filters based on DB name or DB type.
+- [PMM-11832](https://jira.percona.com/browse/PMM-11832) - When hovering over the **Backup** option on the main menu, PMM now displays the subcategories for easier navigation.
+- [PMM-11522](https://jira.percona.com/browse/PMM-11522) - With the newly added support for Clickhouse as a datasource for Advisors, Advisors can now use data produced by Query Analytics (QAN) to run checks on databases.
```

Comment:
BTW @artemgavrilov do advisors support external clickhouse?

Reply:
It should, we are using DSN from PMM server configuration (flags/envs)